js/symb/leafletMap.js

```diff
@@ -0,0 +1,201 @@
+const DEFAULT_DRAW_OPTIONS = {
+   polyline: false,
+   control: true,
+   circlemarker: false,
+   marker: false,
+   multiDraw: false,
+   drawColor: { color: '#000', opacity: 0.85, fillOpacity: 0.55 }
+};
+
+const DEFAULT_MAP_OPTIONS = {
+   center: [43.64701, -79.39425],
+   zoom: 15,
+};
+
+const DEFAULT_SHAPE_OPTIONS = {
+   color: '#000',
+   opacity: 0.85,
+   fillOpacity: 0.55
+};
+
+class LeafletMap {
+   /* To Hold Reference to Leaflet Map */
+   mapLayer;
+
+   /* Active_Shape Type
+    * type: String represenation of shape 
+    * layer: leaflet layer
+    * center: { lat: float, lng float }
+    */
+   activeShape;
+
+   /* Reference Leaflet Feature Group for all drawn items*/
+   drawLayer;
+
+   constructor(map_id, map_options=DEFAULT_MAP_OPTIONS) {
+      this.mapLayer = L.map(map_id, map_options);
```

Comment:
`L` here is coming from the leaflet library?

Reply:
It is a convention but it is also the class name so there really isn't anything to be done other than a wrapper class but that seems like it would be more confusing